packages/enzyme-adapter-react-13/src/ReactThirteenAdapter.js

```diff
@@ -271,6 +271,10 @@ class ReactThirteenAdapter extends EnzymeAdapter {
     return typeof object === 'string' || typeof object === 'function';
   }
 
+  isCustomComponentElement(inst) {
+    return !!inst && this.isValidElement(inst) && typeof inst.type === 'function';
```

Comment:
for now, let's have every adapter where it matches, directly use the implementation from utils